OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/triggers/impl/DynamicTriggerController.kt

```diff
@@ -44,14 +44,31 @@ internal class DynamicTriggerController(
             }
 
             var currentTimeInterval: Long = 0
+            val lastTimeAppDismissed = _state.lastTimeInAppDismissed
```

Comment:
`lastTimeAppDismissed` is missing `In` in the name, should be `lastTimeInAppDismissed`

Reply:
Outdated due to rework